spec/controllers/landing_applications_controller_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+RSpec.describe Officer::LandingApplicationsController do
+  describe "GET to :index" do
+    it "retrieves the landing applications and landable bodies from the database" do
+      allow(LandingApplication).to receive_message_chain(:all, :order)
+      allow(LandableBody).to receive(:all)
+
+      get :index
+
+      expect(LandingApplication).to have_received(:all)
+      expect(LandingApplication.all).to have_received(:order).with(application_submitted_at: :desc)
+      expect(LandableBody).to have_received(:all)
+    end
```

Comment:
you can now rework this slightly to describe the way in which we:

- eager load the `:destinations` with `includes()`
- order the landing application newest first

something like:

```rb
    it "retrieves the landing applications, newest first, with destinations loaded" do
      ar_query = double('query', order: double)
      allow(LandingApplication).to receive(:includes).and_return(ar_query)

      get :index

      expect(LandingApplication).to have_received(:includes).with(:destination)
      expect(ar_query).to have_received(:order).with(application_submitted_at: :desc)
    end
```